app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -87,6 +87,91 @@ void setAliveAt_When1_1IsPassed_ReturnsTheCellObjectAt1_1() {
 
     }
 
+    @Nested
+    @DisplayName("willCellSurviveAt()")
```

Comment:
We should figure out how to mock out the `Rules` class here so that the `Board` specs aren't testing both the Board class and the Rules class.

Reply:
For example, a Board could take which Rules to apply as a dependency.

Reply:
I'd like to go over this more in the IPM if we have time. My thought process behind this was to use the testing convention from JB rainsbergers 'integrated tests are a scam' but now I realize I would need an interface to keep the tests as unit tests